llvm/include/llvm/TableGen/Main.h

```diff
@@ -14,7 +14,6 @@
 #define LLVM_TABLEGEN_MAIN_H
 
 #include "llvm/Support/CommandLine.h"
-#include <functional>
```

Comment:
`std::function_ref` is defined in `<functional>`, why is this header being removed?

Reply:
std::function_ref is from c++26, this is llvm::function_ref